pkg/cmdutil/config/repositories.go

```diff
@@ -3,14 +3,17 @@ package config
 import (
 	"errors"
 	"fmt"
+
 	"github.com/coreeng/corectl/pkg/git"
+	"github.com/rs/zerolog/log"
 )
 
-func ResetConfigRepositoryState(repositoryParam *Parameter[string]) (*git.LocalRepository, error) {
+func ResetConfigRepositoryState(repositoryParam *Parameter[string], dryRun bool) (*git.LocalRepository, error) {
+	log.Trace().Msgf("Resetting config repo state")
```

Comment:
Can we also log what repo is being reset?

Reply:
The logging now happens within git.go instead